src/sessions/base-session-handler.ts

```diff
@@ -140,6 +139,14 @@ export default abstract class BaseSessionHandler {
     });
   }
 
+  async setConversationHeld (session: IExtendedMediaSession, params: IConversationHeldRequest): Promise<any> {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=8384644)
```suggestion
  async setConversationHeld (session: IExtendedMediaSession, params: IConversationHeldRequest): Promise<unknown> {
```